skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -303,6 +320,56 @@ def test_init(classifier, config_json):
         )
 
 
+@pytest.fixture(
+    params=[pytest.param("classifier", marks=pytest.mark.usefixtures), get_classifier()]
+)
+def classifiers(request):
+    # Returns a model object or a path to a model with all
+    # model formats combinations from CONFIG dict
+    try:
+        yield request.getfixturevalue(request.param)
+    except Exception:  # get_classifier() is not a fixuture, exception raised
+        yield request.param
+
+
+def test_override_init_modelcard(classifiers, config_json):
+    # create a temp directory and delete it, we just need a unique name.
+    dir_path = tempfile.mkdtemp()
+    shutil.rmtree(dir_path)
+
+    version = metadata.version("scikit-learn")
+    _, model_format = config_json
+    # joblib type falls unders auto format, explicityly set to auto
+    # because we can't repeat key "auto" in CONFIG dict
```

Comment:
Sorry, I don't understand that, could you please explain further?

Reply:
Sorry, that comment isn't self-explainable. The dictionary `CONFIG` in `test_hf_hub` contains 3 types of models. I wanted to test for a model with name `model.joblib`, so I added that model to the dictionary. If I recall correctly, adding the key:value pair for the joblib model made other tests fail. But I could fix this by changing the `joblib` model type `auto` and still have the name `model.joblib` to test for.

Reply:
Hmm, I guess it depends on why the other tests fail. If this uncovers a bug with existing code, that would be good to know. Otherwise, I think it's okay to have a separate test for joblib and not changing `CONFIG`.